modules/repository/variables.tf

```
@@ -0,0 +1,57 @@
+variable "name" {}
+
+variable "supermarket_name_override" {
```

```suggestion
variable "supermarket_name_override" {
type = string
```

Just for completeness we should probably type all the input variables